hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/acl/OzoneNativeAuthorizer.java

```diff
@@ -104,34 +106,45 @@ public boolean checkAccess(IOzoneObj ozObject, RequestContext context)
     // Refined the parent context
     // OP         |CHILD       |PARENT
 
-    // CREATE      NONE         WRITE     (parent:'CREATE' when 'create bucket')
-    // DELETE      DELETE       WRITE
-    // WRITE       WRITE        WRITE
-    // WRITE_ACL   WRITE_ACL    WRITE     (V1 WRITE_ACL=>WRITE)
+    // CREATE      NONE        WRITE     (parent:'CREATE' when 'create bucket')
```

Comment:
Please move this parent ACL logic and comments into a separate function, then both OzoneNativeAuthorizer and OzoneAclUtil can leverage it.

Reply:
@ChenSammi Updated